modules/ROOT/pages/managing-queues.adoc

```diff
@@ -39,17 +46,17 @@ image::persistentqueuessetondeploy.png[PersistentQueuesSetOnDeploy]
 
 == Viewing Queues
 
-After you enable Persistent queues, click *Queues* in the left navigation menu. CloudHub lists the queues in your application in the order that they appear in your application:
+After you enable persistent queues, click *Queues* in the left navigation menu. CloudHub lists the queues in your application in the order they appear in your application:
 
 image::chqconsole.png[CHQConsole]
 
 The name of each queue is determined by the `path` attribute of the associated VM endpoint, or, in the case of a SEDA queue, from the flow name.
 
-The *Queued* column shows the number of messages currently waiting on each of the queues in your application. The *In-Flight* column indicates the number of messages being processed in the flow after that queue. The in-flight messages are technically still on the queue until processing is complete – only then are they deleted from the queue. The *Processed Messages* column shows a thumbnail graph depicting the number of messages processed through each queue over the past 24 hours. The Queued and In-Flight columns update every five seconds, and the Processed Messages graph updates every five minutes.
+The *Queued* column shows the number of messages currently waiting on each of the queues in your application. The *In-Flight* column indicates the number of messages being processed in the flow after that queue. The in-flight messages are still on the queue until processing is complete, then they are deleted from the queue. The *Processed Messages* column shows a thumbnail graph depicting the number of messages processed through each queue over the past 24 hours. The *Queued* and *In-Flight* columns update every five seconds, and the *Processed Messages* graph updates every five minutes.
```

Comment:
Maybe break up into a bulleted list?
**Queued**: Displays the number of messages...
**In-Flight**: Indicates the number of messages...

Reply:
Yep.